runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -992,5 +1190,42 @@ public CorfuRuntime setTimeoutInMinutesForFastLoading(int timeout) {
         parameters.setFastLoaderTimeout(Duration.ofMinutes(timeout));
         return this;
     }
+
+
+    /**
+     * Add a layout server to the list of servers known by the CorfuRuntime.
+     *
+     * @param layoutServer A layout server to use.
+     * @return A CorfuRuntime, to support the builder pattern.
+     * @deprecated This method is deprecated. Please use the {@link CorfuRuntimeParameters} object.
+     */
+    @Deprecated
+    public CorfuRuntime addLayoutServer(@Nonnull String layoutServer) {
+        layoutServers.add(layoutServer);
+        return this;
+    }
+
+    /**
+     * Stop all routers associated with this runtime & disconnect them.
+     *
+     * @deprecated It is no longer possible to "stop" but not "shutdown" the runtime.
+     *             Please call {@link this#shutdown()} instead.
+     */
+    @Deprecated
+    public void stop() {
+        log.warn("stop: Calling deprecated stop() method, call shutdown() instead");
+        stop(false);
+    }
+
+    /**
+     * Stop all routers associated with this runtime & disconnect them.
+     *
+     * @deprecated It is no longer possible to "stop" but not "shutdown" the runtime.
+     *             Please call {@link this#shutdown()} instead.
+     */
+    @Deprecated
+    public void stop(boolean shutdown) {
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1133)